projects/core/src/occ/occ-models/occ-endpoints.model.ts

```diff
@@ -658,5 +658,11 @@ export interface OccEndpoints {
    * @member {string}
    */
   saveCart?: string | OccEndpoint;
+  /**
+   * Endpoint for add quick order to active cart
+   *
+   * @member {string}
+   */
+  addToCart?: string | OccEndpoint;
```

Comment:
Why it's added?

Reply:
no needed, the old version required this part of code - removing